runtime/ddr/blacklist

```diff
@@ -27,6 +27,44 @@ file:<built-in>
 # zlib defines the type 'Byte' which conflicts with java.lang.Byte.
 file:*/zlib/zconf.h
 
-# Ignore types that are troublesome or not required. 
+# Ignore types that are troublesome or not required.
 type:__gnu*
 type:*__va_list*
+type:*vc_attributes*
+type:*<*
+type:tag*
+
+type:*API_WORKING_SET_*
+type:*MINIDUMP_*
+type:*OSVERSIONINFOEXW
+type:*PDH_FMT_COUNTER*
+type:*SAFEARRAY*
+type:*SCOPE_TABLE*
+type:*TIME_ZONE_INFORMATION
+type:*UNWIND_*
+type:*WIN32_FIND_DATAW
+type:pthreadlocinfo
+type:threadlocaleinfostruct
+
+# Ignore one (or all) of sets of types whose names differ only in case
+# (so we don't try to create Java source files whose names differ only
+# in case which isn't possible with case-insensitive file-systems).
+
+type:ADDRINFOW
+type:addrinfoW
+type:BYTE
+type:Byte
+type:byte
+type:classTableEntry
```

Comment:
should fix this one instead of excluding as this type may be useful for debugging

Reply:
I think I misunderstood. There is still a structure included with the same name, just not with this case.

Reply:
Maybe it doesn't matter given this structure is a union of different pointer types.

Reply:
Neither ClassTableEntry nor ClassTableEntry is currently used in DDR code.
* `ClassTableEntry` is defined in TgcInterRegionRememberedSetDemographics.cpp
* `classTableEntry` is defined in KeyHashTable.c

We could fix both, but I propose that be done in a separate PR.

Reply:
The structures don't need to be used in DDR code to be useful. When debugging, the structures are used to dump data found in the core file. Without the structures the data needs to be manually interpreted which would be much more painful and error prone. I'm ok to fix the problem in a separate PR, but suggest it be done before this one is merged.

Reply:
Agreed.